src/components/shared/Navbar/Navbar.tsx

```diff
@@ -13,8 +21,10 @@ const Navbar = () => {
         </Typography>
 
         <div className="flex align-trailing px-10 gap-[5rem]">
-          {buttonLabels.map((label, index) => (
-            <StyledNavbarButton label={label} key = {index} />
+          {Object.entries(buttonLabels).map(([key, value]) => (
+            <Link href={`/#${key}`} key={key}>
+              <StyledNavbarButton label={value} key={key} />
+            </Link>
           ))}
 
           {/* TODO: Implement color theme functionality */}
```

Comment:
Not sure what you mean by color theme functionality — feel free to write a ticket for it though

Reply:
I think the dark/light mode toggle in the top right.
![image](https://github.com/user-attachments/assets/8487044b-2e18-4df7-8d8c-d87bdc2b4d0f)

Reply:
Ah ok — think someone already created a ticket for it. But it's a non-essential feature so I'm down to table it.